### ISSUE TITLE
Documentation Out of Date

### ISSUE THREAD

**Issue by GloverChris** [neovim/neovim#22987]
### Problem

Paths and installation location on Windows differs from documentation. ~/AppData/local/nvim-data is what I am seeing the folder names as rather than nvim. Installation location differs from documentation, installation source is instead installing in C:\tools\neovim\nvim-win64. Tried all installation methods and found the locations are the same and tested on 3 different machines. Can you please use the proper paths for your installation so users can customize the app? Please don't create a folder in root of disk to install your program rather use the proper folder C:\Program Files and for application data specific to the user please use ~/AppData/Local/. Thanks! <3 "hopefully when reading this my tone isn't coming off as rude."

### Steps to reproduce

1) Install neovim : winget install Neovim.Neovim
2) Navigate to AppData and you will see the appdata folder differs from official documentation
3) Navigate to C:\tools youll see the application is not installed in the proper path

### Expected behavior

nvim data folder folder should be : ~/AppData/Local/nvim
Installation path should be the approved program installation paths per Windows best practice guidelines.

"Best Practices for Application Installation" - This article provides guidance on best practices for installing applications on Windows, including recommendations for installation directories: https://docs.microsoft.com/en-us/windows/win32/appinstall/best-practices-for-application-installation

"Managing Application Deployment" - This article provides guidance on managing application deployment on Windows, including recommendations for software installation directories: https://docs.microsoft.com/en-us/windows/win32/dxtecharts/managing-application-deployment

### Neovim version (nvim -v)

v0.8.3

### Vim (not Nvim) behaves the same?

NA

### Operating system/version

Windows 10.0.22621 Build 22621

### Terminal name/version

Windows Terminal Version: 1.16.10262.0

### $TERM environment variable

NA

### Installation

winget, chocolatey

**Comment by glacambre** [comment]
>v0.8.3

Your version of Neovim is out of date, the latest stable version is v0.9.0 :).

>Paths and installation location on Windows differs from documentation.

Could you describe exactly which page of the Neovim documentation differs from what it actually does? https://neovim.io/doc/user/starting.html#standard-path mentions using `nvim-data`, so everything looks up to date to me.

**Comment by dundargoc** [comment]
Let's back up a bit. That the neovim executable somehow ends up in `C:\tools` and that the data directory is called `nvim-data` are separate issues. That the data directory is called `nvim-data` is a conscious decision, but I don't remember all of the arguments as to why it was created at the top of my head. It is alright if you disagree that the data dir is called `nvim-data`, but please open a new issue where you address that only so that we can have a focused discussion.

As for the `c:\tools` thing: I cannot recreate this at all. You mentioned version 0.8.3, but the latest on winget is 0.9.0. Can you try that and see if it works as expected?

**Comment by GloverChris** [comment]
Hmm...ok I think I made an oopsie here as I was following some information based on a tutorial I was following and NeoVim's docs are necessarily out of date but rather the information I as following is. First off, I'd like to apologize for that mistake and secondly, I have some follow up questions plus wanted to provide some of the material I was referencing that caused me confusion. Also want to acknowledge the request to cut an issue regarding the installation location of nvim data files and let you know that I'll do that. As for the installation location situation I'll try to grab the latest version from the RPM and if that doesn't work Ill build from source.


Reference Material I was following: https://github.com/nvim-lua/kickstart.nvim

Question: 
1) When I follow the instructions [1] to install nvim on Windows I seem to get the older version that was spotted in the comments above. Is that expected?

2) Is there a link to the discussion that was had about the data naming location of NeoVim on Windows? Wanted to see the logic behind the situation for my own learning. 


[1] https://github.com/neovim/neovim/wiki/Installing-Neovim



Just want to say that I appreciate all your time and effort on this project. I try to say this because maintaining a repository can be a thankless job and wanted to show that I appreciate you. ❤️

**Comment by dundargoc** [comment]
> When I follow the instructions [1] to install nvim on Windows I seem to get the older version that was spotted in the comments above. Is that expected?

No, absolutely not. If you run `winget show neovim.neovim` you should see that it says version 0.9.0. If you install that, and still get version v0.8.3 then I suspect you may need to remove/uninstall the 0.8.3 version first to make sure the new neovim is found. Can you try purging your system of all versions of neovim, and the installing it with `winget install neovim.neovim`?

> Is there a link to the discussion that was had about the data naming location of NeoVim on Windows? Wanted to see the logic behind the situation for my own learning.

Probably, but I don't have the time at the moment. I'm pretty sure it's because Neovim follows XDG Base Directory Specification, but since Windows doesn't have that then this is an OK approximation. See `:help xdg`. If you dislike it you can change the locations by modifying the environment variables `$XDG_DATA_HOME` and `$XDG_STATE_HOME`.

**Comment by GloverChris** [comment]
I believe I found the culprit. It is not WinGet but rather Chocolatey consistently installs the older package after purge. Checking chocolatey repo for the package it looks like your update is failing [1] and waiting on maintainer actions

choco list -lo -a neovim                                                                                                                                                                
Chocolatey v1.2.1
neovim 0.8.3
1 packages installed.

[1]https://community.chocolatey.org/packages/neovim#versionhistory

**Comment by dundargoc** [comment]
We don't maintain the chocolatey package. If it installs somewhere where it shouldn't then it's not something we can really prevent unfortunately. Again, please try removing all versions of neovim from your computer and then install with winget please.

**Comment by GloverChris** [comment]
Yep yep, I shall use WinGet moving forward. Chocolatey has been the standard for NT devs and power users for years and WinGet is still new so some of us are in the transition phase with it still. Anyhow, thanks for the conversation, I appreciate your time.

**Comment by dundargoc** [comment]
I personally prefer scoop on windows, but they should all work (emphasis on *should* :) )

**Comment by GloverChris** [comment]
I have never used scoop. Im going to spend some time to look into it :)
